pages/learn/accounts/enterprise-sso.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Enterprise Single Sign-On
+excerpt: Setup balenaCloud SSO authentication using SAML 2.0
+---
+
+# Enterprise Single Sign-On (SSO)
+
+__Note:__ This feature is currently available only on [Enterprise plans](https://www.balena.io/pricing).
+
+BalenaCloud Enterprise Single Sign-On (SSO) using SAML (Security Assertion Markup Language) allows organizations to manage user access and authentication through their existing Identity Providers (IdP). This integration enables users to use their corporate credentials to log in and access BalenaCloud services. By leveraging SAML, enterprises can simplify the login process, enhance security, streamline user management, and ensure compliance with their internal policies and procedures.
+
+Configuring an Identity Provider (IdP) as a login method requires a one-time setup process within both balenaCloud and the identity provider itself. Refer to our [IdP specific documentation][ms-saml] for detailed instructions on required configurations.
+
+## Link a SAML Identity Provider
+
+To enable Single Sign-On (SSO) for balenaCloud organizations, you must establish a connection with your external Identity Provider (IdP). This process assumes that you have already configured a [SAML 2.0 IdP and possess an XML certificate][ms-saml] ready for upload.
```

Comment:
I think we need to make a little bit more clear that we support all SAML 2.0 Identity providers, we just showed the examples of how to do with MS/and google workspace as they are more common but any SAML 2.0 IdP is supported (okta, etc)